homeassistant/components/sensor/teksavvy.py

```diff
@@ -0,0 +1,168 @@
+"""
+Support for TekSavvy.
+
+Get bandwidth comsumption data using Teksavvy API
+You can get your API key here:
+https://myaccount.teksavvy.com/ApiKey/ApiKeyManagement
+
+TekSavvy only counts download only as part of the bandwidth
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.teksavvy/
+"""
+import logging
+from datetime import timedelta
+import asyncio
+import async_timeout
+import voluptuous as vol
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.const import (
+    CONF_API_KEY, CONF_NAME, CONF_MONITORED_VARIABLES)
```

Comment:
Please keep the import groups and sorted. Use `isort` if you don't want to do it manually.

Reply:
Done. Thanks for the isort tip.